### ISSUE TITLE
no licensing information?

### ISSUE THREAD

**Issue by moedn** [OPEN-NEXT/LOSH-Appropedia-Scraper#1]
https://gitlab.opensourceecology.de/verein/projekte/losh-rdf/-/raw/v0.1/RDF/appropedia.org/Backpack_frame_bike_trailer/0_0/okh.ttl

does not contain any licensing information although [Appropedias original page](https://www.appropedia.org/Backpack_frame_bike_trailer) does. that would be a crucial bug :)

Do we have a fallback if no license is found? I believe Appropedia has a standard licensing scheme if users don't specify any other

**Comment by hoijui** [comment]
This should be solved if #2 is fixed, as we are simply using outdated manifests.

The generated OKH v1 for that project can be found here:
https://www.appropedia.org/okh.php?title=Backpack_frame_bike_trailer

and does contain the license info! :-)

```
#License
license: 
  documentation: CC BY-SA 3.0
licensor:
  name: Aaron Antrim
  contact: https://www.appropedia.org/Aaron
documentation-home: https://www.appropedia.org/Backpack_frame_bike_trailer
```

So we can scrape it then!

On the old page, the license was not scrape-able in the same way as the other meta-data.

**Comment by hoijui** [comment]
Fixed with f4c0898, as the OKH files generated by Appropedia themselves now, do contain licensing info (and also more other properties)! :-)
